### PR TITLE
Elastic Defend event capture explainer

### PR DESCRIPTION
**Pull request by joepeeples**
Contributes to https://github.com/elastic/security-docs/issues/5028.

### Preview
- [Event capture and Elastic Defend](https://security-docs_bk_5194.docs-preview.app.elstc.co/guide/en/security/master/endpoint-event-capture.html) - New page

### Twin PR for serverless
TBD, once content is fully finalized in this PR.

### CODE REVIEW

**Review by gabriellandau — APPROVED** [review]
Thanks for adding this to docs.

**Review by ferullo — COMMENTED** [review]
I'll hold off on giving an LGTM because there's lots of technical info in here I can't review. But this doc looks good overall, I assume it's ok to publish it and then add to it over time.

**Review by caitlinbetz — DISMISSED** [review]

File: docs/management/admin/endpoint-event-capture.asciidoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+== Network port creation and deletion
+
+{elastic-defend} tracks TCP connections. If a port is created but no traffic flows, no events will be generated.
+
+For complete capture of network port creation and deletion, consider capturing Windows event ID 5158 using the {integrations-docs}/winlog[Custom Windows Event Logs] integration.
+
+[discrete]
+== Network in/out connections
+
+{elastic-defend} tracks TCP connections, which don't include network in/out connections.
```
Comment (nastasha-solomon) [inline docs/management/admin/endpoint-event-capture.asciidoc:20]: Optional edit for clarity.

```suggestion
{elastic-defend} tracks TCP connections, which don't include network in or out connections.
```

File: docs/management/admin/endpoint-event-capture.asciidoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+{elastic-defend} collects selective data on system activities in order to detect and prevent as many threats as possible, while balancing storage and performance overhead. To that end, {elastic-defend} isn't designed to provide a complete capture of all system events. The event data that {elastic-defend} generates might be aggregated, truncated, or deduplicated as needed to optimize threat detection and prevention.
+
+You can supplement {elastic-defend}'s protection capabilities with additional {integrations-docs}[Elastic integrations] and tools that provide more visibility and historical data. Consult the following sections to expand data collection for specific system events.
+
+
+[discrete]
+== Network port creation and deletion
+
+{elastic-defend} tracks TCP connections. If a port is created but no traffic flows, no events will be generated.
```
Comment (nastasha-solomon) [inline docs/management/admin/endpoint-event-capture.asciidoc:13]:
```suggestion
{elastic-defend} tracks TCP connections. If a port is created but no traffic flows, no events are generated.
```

**Review by nastasha-solomon — APPROVED** [review]
Good job synthesizing this! I left two edits that are completely optional if you need to merge this ASAP.

File: docs/management/admin/endpoint-event-capture.asciidoc (hunk 1 above)
Comment (joepeeples) [reply docs/management/admin/endpoint-event-capture.asciidoc:20]: Seems like "in/out" is a sorta common IT term so probably keep this as is.

**Review by nastasha-solomon — APPROVED** [review]

**Pull request merged**